### PR TITLE
docs: changelog for 5.16.2

### PR DESCRIPTION
**Pull request by PeachScript**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- 🐞 Fix Input.OTP controlled `value` to `''` not work as expect. [#48399](https://github.com/ant-design/ant-design/pull/48399)
- 🐞 Fix DatePicker of `multiple` with week picker not show the selection on the panel. [#48355](https://github.com/ant-design/ant-design/pull/48355)
- 🐞 Fix Upload `listType` with `picture-card` or `picture-circle` has additional space on top. [#48370](https://github.com/ant-design/ant-design/pull/48370)
- Typography
  - 🐞 Fix Typography dynamic set `copyable` or `editable` will not show the operation button. [#48350](https://github.com/ant-design/ant-design/pull/48350)
  - 🐞 Fix Typography dynamic update `copyable.text` not working. [#48347](https://github.com/ant-design/ant-design/pull/48347)
  - 🐞 Fix Typography using `ellipsis` config with `expandable=collapsible` and `row=1` at both time will make ellipsis not working as expect. [#48340](https://github.com/ant-design/ant-design/pull/48340)
- 💄 Fix Steps in compact theme not show the correct progress style. [#48251](https://github.com/ant-design/ant-design/pull/48251)
- 💄 Use `motionDurationSlow` Component Token instead of related css property values. [#48312](https://github.com/ant-design/ant-design/pull/48312) [@li-jia-nan](https://github.com/li-jia-nan)
- 🌐 Fix Picker i18n translations. [#48382](https://github.com/ant-design/ant-design/pull/48382) [@li-jia-nan](https://github.com/li-jia-nan)
- 🛠 Use constant instead of enumeration to reduce package size. [#48406](https://github.com/ant-design/ant-design/pull/48406) [@kiner-tang](https://github.com/kiner-tang)

---

- 🐞 修复 Input.OTP 组件受控设置 `value` 为 `''` 时不生效的问题。[#48399](https://github.com/ant-design/ant-design/pull/48399)
- 🐞 修复 DatePicker 对周选择器配置 `multiple` 多选时，弹出面板不会正确高亮选中项的问题。[#48355](https://github.com/ant-design/ant-design/pull/48355)
- 🐞 修复 Upload 配置 `listType` 为 `picture-card` 或 `picture-circle` 时，上方有额外空隙的问题。[#48370](https://github.com/ant-design/ant-design/pull/48370)
- Typography
  - 🐞 修复 Typography 动态配置 `copyable` 或 `editable` 时不会显示操作按钮的问题。[#48350](https://github.com/ant-design/ant-design/pull/48350)
  - 🐞 修复 Typography 动态调整 `copyable.text` 不生效的问题。[#48347](https://github.com/ant-design/ant-design/pull/48347)
  - 🐞 修复 Typography 的 `ellipsis` 同时配置 `expandable=collapsible` 和 `row=1` 时，不会正确省略的问题。[#48340](https://github.com/ant-design/ant-design/pull/48340)
- 💄 修复 Steps 的进度样式在紧凑模式下不正确的问题。[#48251](https://github.com/ant-design/ant-design/pull/48251)
- 💄 使用 `motionDurationSlow` Component Token 替换相关 CSS 属性值。[#48312](https://github.com/ant-design/ant-design/pull/48312) [@li-jia-nan](https://github.com/li-jia-nan)
- 🌐 修复 Picker 国际化语言翻译错误。[#48382](https://github.com/ant-design/ant-design/pull/48382) [@li-jia-nan](https://github.com/li-jia-nan)
- 🛠 使用常量替代枚举减小包体积。[#48406](https://github.com/ant-design/ant-design/pull/48406) [@kiner-tang](https://github.com/kiner-tang)

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by kiner-tang — DISMISSED** [review]

**Comment by li-jia-nan** [comment]
@SocketSecurity ignore npm/sharp@0.33.3
@SocketSecurity ignore npm/puppeteer@22.6.4

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
...
+
+`2024-04-13`
+
+- 🐞 修复 Input.OTP 组件受控设置 `value` 为 `''` 时不生效的问题。[#48399](https://github.com/ant-design/ant-design/pull/48399)
+- 🐞 修复 DatePicker 对周选择器配置 `multiple` 多选时，弹出面板不会正确高亮选中项的问题。[#48355](https://github.com/ant-design/ant-design/pull/48355)
+- 🐞 修复 Upload 配置 `listType` 为 `picture-card` 或 `picture-circle` 时，上方有额外空隙的问题。[#48370](https://github.com/ant-design/ant-design/pull/48370)
+- Typography
+  - 🐞 修复 Typography 动态配置 `copyable` 或 `editable` 时不会显示操作按钮的问题。[#48350](https://github.com/ant-design/ant-design/pull/48350)
+  - 🐞 修复 Typography 动态调整 `copyable.text` 不生效的问题。[#48347](https://github.com/ant-design/ant-design/pull/48347)
+  - 🐞 修复 Typography 的 `ellipsis` 同时配置 `expandable=collapsible` 和 `row=1` 时，不会正确省略的问题。[#48340](https://github.com/ant-design/ant-design/pull/48340)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:29]:
```suggestion
  - 🐞 修复 Typography 的 `ellipsis` 同时配置 `expandable="collapsible"` 和 `row={1}` 时，不会正确省略的问题。[#48340](https://github.com/ant-design/ant-design/pull/48340)
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
...
+- 💄 修复 Steps 的进度样式在紧凑模式下不正确的问题。[#48251](https://github.com/ant-design/ant-design/pull/48251)
+- 💄 使用 `motionDurationSlow` Component Token 替换相关 CSS 属性值。[#48312](https://github.com/ant-design/ant-design/pull/48312) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🌐 修复 Picker 国际化语言翻译错误。[#48382](https://github.com/ant-design/ant-design/pull/48382) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🛠 使用常量替代枚举减小包体积。[#48406](https://github.com/ant-design/ant-design/pull/48406) [@kiner-tang](https://github.com/kiner-tang)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:33]:
```suggestion
- 📦 使用常量替代枚举减小包体积。[#48406](https://github.com/ant-design/ant-design/pull/48406) [@kiner-tang](https://github.com/kiner-tang)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
...
+- 🐞 修复 Input.OTP 组件受控设置 `value` 为 `''` 时不生效的问题。[#48399](https://github.com/ant-design/ant-design/pull/48399)
+- 🐞 修复 DatePicker 对周选择器配置 `multiple` 多选时，弹出面板不会正确高亮选中项的问题。[#48355](https://github.com/ant-design/ant-design/pull/48355)
+- 🐞 修复 Upload 配置 `listType` 为 `picture-card` 或 `picture-circle` 时，上方有额外空隙的问题。[#48370](https://github.com/ant-design/ant-design/pull/48370)
+- Typography
+  - 🐞 修复 Typography 动态配置 `copyable` 或 `editable` 时不会显示操作按钮的问题。[#48350](https://github.com/ant-design/ant-design/pull/48350)
+  - 🐞 修复 Typography 动态调整 `copyable.text` 不生效的问题。[#48347](https://github.com/ant-design/ant-design/pull/48347)
+  - 🐞 修复 Typography 的 `ellipsis` 同时配置 `expandable=collapsible` 和 `row=1` 时，不会正确省略的问题。[#48340](https://github.com/ant-design/ant-design/pull/48340)
+- 💄 修复 Steps 的进度样式在紧凑模式下不正确的问题。[#48251](https://github.com/ant-design/ant-design/pull/48251)
+- 💄 使用 `motionDurationSlow` Component Token 替换相关 CSS 属性值。[#48312](https://github.com/ant-design/ant-design/pull/48312) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🌐 修复 Picker 国际化语言翻译错误。[#48382](https://github.com/ant-design/ant-design/pull/48382) [@li-jia-nan](https://github.com/li-jia-nan)
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:32]:
```suggestion
- 🌐 修复 Picker 国际化日语翻译错误。[#48382](https://github.com/ant-design/ant-design/pull/48382) [@li-jia-nan](https://github.com/li-jia-nan)
```

Comment (afc163) [reply CHANGELOG.zh-CN.md:32]: 还有韩语。

**Comment by PeachScript** [comment]
> 漏了一个：
> 
> 
> 
> - #48311

看了下和 48312 是类似的所以就只写了一条，我加一下 PR 链接

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (PeachScript) [reply CHANGELOG.zh-CN.md:33]: 这个图标过不了 lint
图片

**Comment by li-jia-nan** [comment]
> 看了下和 48312 是类似的所以就只写了一条，我加一下 PR 链接

感觉可以写一下具体的组件名字？万一出了问题，方便将来追溯

**Comment by li-jia-nan** [comment]
> 涉及的组件有哪些

tabs组件和内部水波纹组件

**Comment by yoyo837** [comment]
> > 感觉可以写一下具体的组件名字？万一出了问题，方便将来追溯
> 
> 涉及的组件有哪些

Button、Checkbox、Radio、Switch、Tag

**Comment by li-jia-nan** [comment]
> > 涉及的组件有哪些
> 
> Button、Checkbox、Radio、Switch、Tag

不用这么详细把，写内部波纹组件就好了

**Comment by PeachScript** [comment]
changelog 是给用户看的，所以写内部组件也不合适，我改成 Tabs and wave related components 吧，cc @li-jia-nan @yoyo837

**Comment by li-jia-nan** [comment]
> changelog 是给用户看的，所以写内部组件也不合适，我改成 Tabs and wave related components 吧，cc @li-jia-nan @yoyo837

可以

**Comment by yoyo837** [comment]
> changelog 是给用户看的，所以写内部组件也不合适，我改成 Tabs and wave related components 吧，cc @li-jia-nan @yoyo837

image

我从这里找的，没看到Tabs呢？

**Comment by PeachScript** [comment]
> 没看到Tabs呢

Tabs 在 #48312 里

**Comment by afc163** [comment]
等 https://github.com/ant-design/ant-design/pull/48411 和 https://github.com/ant-design/ant-design/pull/48436 ci 好了合并掉。

**Comment by PeachScript** [comment]
> 等 #48411 和 #48436 ci 好了合并掉。

已合并及更新 changelog

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -16,6 +16,23 @@ tag: vVERSION
 
 ---
 
+## 5.16.2
+
+`2024-04-13`
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:21]:
```suggestion
`2024-04-14`
```

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,23 @@ tag: vVERSION
 
 ---
 
+## 5.16.2
+
+`2024-04-13`
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:20]:
```suggestion
`2024-04-14`
```

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**